src/noui.cpp

```diff
@@ -4,15 +4,18 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include <noui.h>
-
-#include <ui_interface.h>
-#include <util/system.h>
-
+#include <boost/signals2/connection.hpp>
```

Comment:
Why is this line moved?

Reply:
I incorrectly assumed that these imports were alphabetically sorted. This turns out not to be the case. I'll revert this